packaging/commands/apt.go

```diff
@@ -47,7 +47,7 @@ var aptCmder = packageCommander{
 	isInstalled:         buildCommand(dpkgquery, "-s %s"),
 	listAvailable:       buildCommand(aptcache, "pkgnames"),
 	listInstalled:       buildCommand(dpkg, "--get-selections"),
-	addRepository:       buildCommand(addaptrepo, "%q"),
```

Comment:
My mistake, I thought this was a printf statement.

On Wed, Aug 5, 2015 at 8:40 PM, claudiu-coblis <notifications@github.com>
wrote:

> In packaging/commands/apt.go
> <https://github.com/juju/utils/pull/145#discussion_r36288874>:
>
> > @@ -47,7 +47,7 @@ var aptCmder = packageCommander{
> >  	isInstalled:         buildCommand(dpkgquery, "-s %s"),
> >  	listAvailable:       buildCommand(aptcache, "pkgnames"),
> >  	listInstalled:       buildCommand(dpkg, "--get-selections"),
> > -	addRepository:       buildCommand(addaptrepo, "%q"),
>
> This is not going to compile.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/utils/pull/145/files#r36288874>.
>